gtk4/src/subclass/widget.rs

```diff
@@ -1224,6 +1224,7 @@ pub unsafe trait WidgetClassSubclassExt: ClassStruct {
     unsafe fn bind_template_child_with_offset<T>(
         &mut self,
         name: &str,
+        internal: bool,
```

Comment:
This is of course an API change, so can't be backported

Reply:
yeah, this is something I would keep till 0.3, I don't think a lot of people would use such APIs. But it's nice to have it there someday :)